docs/cli/config.md

```diff
@@ -0,0 +1,65 @@
+# Configuration Files
+
+The Sprocket configuration loader sources from command-line options, environment variables, and configuration files, in that priority order.
```

Comment:
```suggestion
The Sprocket configuration loader sources and merges values from a location specified by a command-line option, an environment variable, and common configuration locations, in that priority order.
```
my edit could potentially use some more word smithing but I think is more accurate/clear